org.eclipse.winery.frontends/app/topologymodeler/src/app/services/policy.service.ts

```diff
@@ -0,0 +1,29 @@
+import { EventEmitter, Injectable, Output } from '@angular/core';
+
+/********************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+@Injectable()
+export class PolicyService {
+    @Output() addNewPolicyEvent: EventEmitter<any>;
```

Comment:
Why do you need this service?
Also, the output annotation can only be used in components.